resources/dw/resource_dw_acc_test.go

```diff
@@ -95,8 +102,26 @@ func AwsDataLakePreCheck(t *testing.T) {
 	}
 }
 
-func TestAccCluster_basic(t *testing.T) {
+func PreCheck(t *testing.T) {
+	if os.Getenv(cdp.CdpProfileEnvVar) == "" && os.Getenv(cdp.CdpAccessKeyIdEnvVar) == "" {
+		t.Skipf("Terraform acceptance testing requires either %s or %s environment variables to be set", cdp.CdpProfileEnvVar, cdp.CdpAccessKeyIdEnvVar)
+	}
+
+	if os.Getenv(cdp.CdpAccessKeyIdEnvVar) != "" {
+		if _, ok := os.LookupEnv(cdp.CdpPrivateKeyEnvVar); !ok {
```

Comment:
What if the `CdpPrivateKeyEnvVar` is set but the `CdpAccessKeyIdEnvVar`?

Reply:
That is fine, we only need the profile to be set for sure, I could replace this logic with 
```
if ak, pk := os.Getenv(cdp.CdpAccessKeyIdEnvVar), os.Getenv(cdp.CdpPrivateKeyEnvVar); (ak != "" && pk == "") || (ak == "" && pk != "") {
```